README.md

```diff
@@ -23,6 +25,17 @@ Beatstream is an app for streaming music from your computer to anywhere with a m
     -> exit
     $ rails server -d
 
+#### Using docker
+
+```bash
+$ cp config/initializers/music_paths.rb.sample config/initializers/music_paths.rb
+$ docker compose build app
+$ docker compose run --rm app bundle install
+$ docker compose run --rm app bundle exec rake db:migrate
+$ docker compose run --rm app bundle exec rake db:seed
```

Comment:
Why we decided to run those steps manually instead of putting this in the build?

Reply:
If we put in the dockefile, every time that the Gemfile changes, we need to rebuild the Image. For production, ok, but this dockefile will be used only for dev.

Also, the migrate and seeds cannot be done on build time, they need to run only when is necessary.